LibTest/io/Directory/Directory_A02_t03.dart

```diff
@@ -21,11 +21,12 @@ import "../../../Utils/expect.dart";
 import "../file_utils.dart";
 
 main() {
-  File file = getTempFileSync();
+  Directory sandbox = getTempDirectorySync();
```

Comment:
This pattern repeats a lot - I wonder if it is worth introducing helper like so:

```dart
// sandbox.dart
Future<void> inSandbox(dynamic testMain(Directory sandbox)) async {
  Directory sandbox = getTempDirectorySync();
  try {
    await testMain(sandbox);
  } finally { 
   sandbox.delete(recursive: true);
  }
}
```

```dart
import '../sandbox.dart';

main() => inSandbox(_main);
_main(Directory sandbox) {
  File file = getTempFileSync(parent: sandbox);
  Directory dir = new Directory(file.path);
  Expect.equals(file.path, dir.absolute.path);
}
```

Reply:
I'd like to left is as is. Pattern repeats, but tests became simpler and much more obvious for the reader. If we need to change it "Find in path" option will hepl

Reply:
ok